src/sinks/prometheus/exporter.rs

```diff
@@ -492,7 +486,7 @@ mod tests {
     #[tokio::test]
     async fn sink_absolute() {
         let config = PrometheusExporterConfig {
-            address: PROMETHEUS_ADDRESS_TLS.parse().unwrap(),
+            address: "127.0.0.1:9105".parse().unwrap(),
```

Comment:
Could we add a comment that this one is never actually bound to? I was tripped up by that initially.

Reply:
Done